tests/ast/test_ast_driver.py

```diff
@@ -232,7 +249,7 @@ def run_test(session):
 
 @pytest.mark.parametrize("test_case", load_test_cases(), ids=idfn)
 def test_ast(session, test_case):
-    actual, base64 = run_test(
+    actual, base64_str = run_test(
```

Comment:
base64 is a str by definition, i don't think this name change is good

Reply:
it conflicts with the python module `base64`. I am accessing that particular module below. If you feel strongly, I can do `from base64 import ...` to call these functions and restore the name.

Reply:
Ah, no, that's cool.